index.html

```diff
@@ -2862,14 +2966,14 @@ <h5><code>subscribeallevents</code></h5>
               const lampEventsSource = new EventSource('/things/lamp/events');
             </pre>
           </p>
-          <p class="rfc2119-assertion" 
+	  <p><span class="rfc2119-assertion" 
           id="http-sse-profile-protocol-binding-subscribeallevents-3">
```

Comment:
Duplicate `id`. Not the fault of this PR but easiest to fix here.

Reply:
fixed, second one renumbered to 4